scripts/all-core.sh

```diff
@@ -165,13 +165,19 @@ pre_initialize_variables () {
             PSA_CORE_PATH='tf-psa-crypto/core'
             BUILTIN_SRC_PATH='tf-psa-crypto/drivers/builtin/src'
             CONFIG_TEST_DRIVER_H='tf-psa-crypto/tests/configs/crypto_config_test_driver.h'
+            MBEDTLS_ROOT_DIR="$PWD"
+            TF_PSA_CRYPTO_ROOT_DIR="$PWD/tf-psa-crypto"
+            MBEDTLS_FRAMEWORK_ROOT_DIR="$PWD/framework"
```

Comment:
Question: we already have a variable `FRAMEWORK` defined in the top-level file (`all.sh`), why do we need that one in addition?

Reply:
I believe that I added this before it was finalised where FRAMEWORK was going to be set as a temporary workaround. I can remove that now.